extras/raspimjpeg.py

```diff
@@ -228,13 +222,23 @@ def init_camera():
     if options.colfx is not None:
         camera.color_effects = options.colfx
     
+    if options.preview:
+	camera.start_preview()
+
     logging.debug('camera initialized')
 
 
+
+def streams_iter():
+
+    while running:
+	yield sys.stdout
+	sys.stdout.flush()
+
+
 def run():
     logging.debug('starting capture')
-    camera.capture_sequence(streams_iter(), format='jpeg',
-            use_video_port=not options.stills, thumbnail=None, quality=options.quality)
+    camera.capture_sequence(streams_iter(), format='jpeg', resize=(options.s_width, options.s_height), use_video_port=not options.stills, thumbnail=None, quality=options.quality)
```

Comment:
Need to split this into two lines, to reduce page width. Sorry, missed this before commit